js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -2143,11 +2161,36 @@ function processQuery(query, explain, planIndex) {
     }
   };
 
-  var postHandle = function (node) {
+  const postHandle = function (node) {
+    if (node.type === 'JoinNode') {
+      ++level;
+      node.indexInfos.forEach((info) => {
+        let line = nodePrefix(node);
+        if (profileMode) {
+          // TODO
+          /*
+          line += pad(1 + maxCallsLen - String(node.calls).length) + value(node.calls) + '   ' +
+            pad(1 + maxItemsLen - String(node.items).length) + value(node.items) + '   ' +
+            pad(1 + maxFilteredLen - String(node.filtered).length) + value(node.filtered) + '   ' +
+            pad(1 + maxRuntimeLen - runtime.length) + value(runtime) + '   ';
+            */
```

Comment:
Profiling output doesn't work yet. I think it is not hard to fix this. As our aim is to run performance tests with the JoinNode, I would be for adding support for profiling output to this PR.